src/metatensor/models/utils/loss.py

```diff
@@ -0,0 +1,118 @@
+from typing import Dict, Optional
+
+import torch
+from metatensor.torch import TensorMap
+
+
+# This file defines losses for metatensor models.
+
+
+class TensorMapLoss:
```

Comment:
This should be added to the docs as well, right?

Reply:
No, it is to be moved to MT-learn (see PR comment at the top), so I think it's pointless to add them to the docs here